marathon_acme/tests/test_service.py

```diff
@@ -320,7 +320,8 @@ def test_sync_app_multiple_ports(self):
             'portDefinitions': [
                 {'port': 9000, 'protocol': 'tcp', 'labels': {}},
                 {'port': 9001, 'protocol': 'tcp', 'labels': {}}
-            ]
+            ],
+            'ports': [10007]
```

Comment:
This is implicit and messy.. I'm trying to check that `portDefinitions` is preferred ahead of `ports` but this is not the best way to state that...